consensus/tests/integration_test.rs

```diff
@@ -335,3 +332,108 @@ async fn single_server_propose_1() {
 
     // Todo: verify finalization proofs
 }
+
+#[tokio::test(flavor = "multi_thread")]
+async fn single_seperate_server_propose_1() {
+    setup_test();
+    let (network_id, dms_key) = get_network_id_and_dms_key("single_seperate_server_propose_1");
+
+    let voting_powers = vec![1, 1, 1, 1, 1];
+    let num_nodes = voting_powers.len();
+    let params = ConsensusParams {
+        timeout_ms: 60 * 1_000, // 1 minute
+        repeat_round_for_first_leader: 100,
+    };
+    let round_zero_timestamp = get_timestamp();
+
+    let (server_config, other_configs, peers) =
+        setup_server_client_nodes(network_id.clone(), num_nodes - 1).await;
+
+    let block_header = configs_to_block_header(
+        once(&server_config).chain(&other_configs).collect(),
+        voting_powers,
+    );
+
+    // Create active nodes (client nodes) that will be executed by CLI.
```

Comment:
The term 'active' is not meaningful IMO.

Reply:
removed it